packages/lest/package.json

```diff
@@ -19,8 +19,7 @@
   },
   "scripts": {
     "bundle": "luabundler bundle src/lua/lest.lua -p \"./?.lua\" -o dist/lua/lest.lua",
-    "minify": "lua scripts/minify.lua",
-    "build:lua": "npm run bundle && npm run minify",
+    "build:lua": "npm run bundle",
```

Comment:
Does the CI call the `bundle` script? If not can we merge them into just `build:lua`?

Reply:
Nope, CI is completely oblivious to all of our details related to actual packaging, it just calls `npm run build:lest` from the root dir